src/constants/ecosystem.ts

```diff
@@ -0,0 +1,11 @@
+export const DIVERGENT_CATEGORY_MAP = {
+  Community: ["Community", "DAO", "Governance"],
+  DeFI: ["DEX", "DeFI", "Launchpad", "Lending", "Marketplace", "Payment"],
```

Comment:
@Holybasil @sharkqwy can we still change the typo `DeFI` to `DeFi`, please? (it will only be harder in the future)

Reply:
please contact me after replacing `DeFI` with `DeFi` in notion db @sharkqwy

Reply:
gentle bump, @sharkqwy